front/components/sparkle/AppContentLayout.tsx

```diff
@@ -71,6 +74,21 @@ export default function AppContentLayout({
     setLoaded(true);
   }, []);
 
+  // Temporary measure to preserve title existence on smaller screens.
+  const contentWithTitle = React.useMemo(() => {
```

Comment:
This memoization won't work unless all the `children` component we pass to AppContentLayout is also memoized no 🤔?